### ISSUE TITLE
PDF thumbnails sometimes all black

### ISSUE THREAD

**Issue by ahayes** [GCRC/nunaliit#406]
Apparently PDF files that use transparency can sometimes appear all black in the thumbnail. Adding the '- flatten' argument to imagemagick may help with this. (Also... the command line option to imagemagick does not seem to be present in multimedia.properties.)

e.g.
`convert -colorspace sRGB string.pdf -scale 100x100 -background white -flatten string.jpg`

See: http://www.imagemagick.org/discourse-server/viewtopic.php?t=20150

**Comment by boikle** [comment]
I found a potential solution to this issue on [stackoverflow.](http://stackoverflow.com/questions/10934456/imagemagick-pdf-to-jpgs-sometimes-results-in-black-background) 

Adding the ImageMagick command line options -alpha remove or -alpha flatten appears to fix this issue. 

I tested both command line options through the multimedia.properties file and each solution worked. 

Note: tested with the latest build 2.2.8 of Nunaliit, using a pdf document which generates a black thumbnail with the default settings.
